### ISSUE TITLE
Throwaway variable triggers SA1312 (Variable names should begin with lower-case letter)

### ISSUE THREAD

**Issue by mikernet** [DotNetAnalyzers/StyleCopAnalyzers#3170]
If the underscore throwaway variable name `_` is used I don't think it should trigger this rule, i.e:

```C#
        private void EnsureExists()
        {
            var _ = Attributes;
        }
```

**Comment by sharwell** [comment]
The discard syntax does not require the `var` keyword:

```csharp
_ = Attributes;
```

**Comment by sharwell** [comment]
Duplicate of #3057

**Comment by mikernet** [comment]
***facepalm*** Thanks :)
